sleap/gui/commands.py

```diff
@@ -3602,6 +3646,23 @@ def open_website(url: str):
     QtGui.QDesktopServices.openUrl(QtCore.QUrl(url))
 
 
+class AddInstanceGroup(AppCommand):
+    @staticmethod
+    def do_action(context: "CommandContext", params: dict):
+        group_name = params.get("group_name", "New Group")  
+        new_group = InstanceGroup(name=group_name)
+        context.app.instance_groups.append(new_group)
+        context.app.refresh_ui()
+        print(f"Added new instance group: {new_group.name}")
+
+    @staticmethod
+    def ask(context: "CommandContext", params: dict):
+        group_name = context.ui.prompt_user_input("Enter the name for the new instance group:")
+        if group_name:
+            params["group_name"] = group_name
+            return True
+        return False  
+
```

Comment:
You already have an `AddInstanceGroup` class above which I prefer since it doesn't pop-up with a GUI everytime.
```suggestion
```